pywb/static/wombat.js

```diff
@@ -3583,15 +3583,26 @@ var _WBWombat = function($wbwindow, wbinfo) {
                 return;
             }
 
-            var message = {
-                       "url": $wbwindow.WB_wombat_location.href,
-                       "ts": wbinfo.timestamp,
-                       "request_ts": wbinfo.request_ts,
-                       "is_live": wbinfo.is_live,
-                       "title": $wbwindow.document ? $wbwindow.document.title : "",
-                       "readyState": $wbwindow.document.readyState,
-                       "wb_type": "load"
-            }
+            var icons = [];
```

Comment:
Just a quick optimization check.

Since this runs for all iframes, I think probably should add a check for top frame to avoid looping over icons for every iframe. The check is already part of send_top_message, but since we're also looping over all icons, would be good to quickly exit here if not in top frame.

Can just add:

```
Since this runs for all iframes, I think probably makes sense to add:

```
        if (!$wbwindow != $wbwindow.__WB_replay_top) {
            return;
        }
```

Also can you fix the indent to be consistent with rest of wombat if possible? (4 spaces)